src/importexport/musicxml/internal/musicxml/import/importmusicxml.cpp

```diff
@@ -20,28 +20,110 @@
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
 
-/**
- MusicXML import.
- */
-
 #include "global/translation.h"
+
+#include "modularity/ioc.h"
+#include "global/iinteractive.h"
```

Comment:
It looks like the `#ifndef MUSICXML_NO_INTERACTIVE` got lost here. I think that's actually fine; I don't see how that macro could be useful and it seems obsolete because it's never defined. So as far as I'm concerned you can decide to get rid of all these `#ifndef MUSICXML_NO_INTERACTIVE`.

Reply:
Never mind, it _is_ defined under certain circumstances, in CMake (my search settings were wrong). And it's necessary for "Build Without Qt", apparently.